webhook_server/tests/test_github_api.py

```
@@ -736,15 +736,16 @@ def test_prepare_log_prefix_with_color_file(
             assert result2 is not None
 
     @pytest.mark.asyncio
-    async def test_process_check_run_event(self, minimal_hook_data: dict, minimal_headers: dict, logger: Mock) -> None:
+    async def test_process_check_run_event(
+        self, minimal_hook_data: dict, minimal_headers: Headers, logger: Mock
```

Ruff (0.14.10)

Unused method argument: `minimal_hook_data`

(ARG002)